src/components/TodoItem.vue

```diff
@@ -0,0 +1,26 @@
+<template>
+  <li :id="todoId" class="todo">
+      <label>{{todoTitle}}</label>
+      <button @click="$emit('remove', todo.id)">
+      X
+    </button>
+  </li>
+</template>
+
+<script>
+
+import Vue from 'vue'
+
+export default Vue.component('TodoItem', {
+  props: ['todo'],
+  data: function() {
+    return {
+      todoTitle: this.todo.title,
```

Comment:
bag approach  to use  data that base on props, if props will change, data will be the same, use computed properties or use *as Is* in template